packages/lib/src/components/buttons/InfoButtonComponent.wc.svelte

```diff
@@ -33,28 +41,45 @@
         tooltipOpen = false;
     };
 
-    const displayQueryInfo = (): void => {
+    const displayQueryInfo = (queryItem?: AstBottomLayerValue): void => {
         if (showQuery) {
-            message =
-                getHumanReadableQuery().length > 0
-                    ? [getHumanReadableQuery()]
-                    : [noQueryMessage];
+             if (onlyChildInfo) {
+                let childMessage =
+                    buildHumanReadableRecursively(
+                    returnNestedValues(queryItem), "");
```

Comment:
`returnNestedValues()` can also return null, that's why typescript is complaining here. In our case it's not expected to be null, so we can cast with `as AstElement`